couler/argo.py

```diff
@@ -152,6 +153,10 @@ def _dump_yaml():
         print(yaml_str)
 
 
+def create_artifact(path, is_global=False):
```

Comment:
We added this function since there didn't seem to be a way to just add regular artifacts, example generated YAML:
```yaml
  outputs:
      artifacts:
        - name: output-io-3fccdce7
          path: /mnt/output/model
```